cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -332,29 +332,7 @@ function enableInstantSearch(config) {
                         if (item.price && item.price[algoliaData.currencyCode] !== null) {
                             item.price = item.price[algoliaData.currencyCode]
                         }
-                        // If no promotionalPrice, use the pricebooks to display the strikeout price
-                        if (!item.promotionalPrice &&
-                            item.pricebooks &&
-                            item.pricebooks[algoliaData.currencyCode] &&
-                            item.pricebooks[algoliaData.currencyCode].length > 0
-                        ) {
-                            const prices = item.pricebooks[algoliaData.currencyCode].filter(pricebook => {
-                                if (pricebook.onlineFrom && pricebook.onlineFrom > Date.now()) {
-                                    return false;
-                                }
-                                if (pricebook.onlineTo && pricebook.onlineTo < Date.now()) {
-                                    return false;
-                                }
```

Comment:
I've already said it in the past: _you need to understand what the existing code is doing before modifying or deleting it._

This permits to not take into account pricebooks that are not online anymore.
With your update, if I have now a record and a pricebooks that ended yesterday, the strikeout price still display it.
Please try yourself, put this in one of your records:

```
"pricebooks": {
    "USD": [
      {
        "price": 500,
        "onlineFrom": 1733097600000, // 2024-12-02T00:00:00.000Z
        "onlineTo": 1733320800000, // 2024-12-04T14:00:00.000Z
        "pricebookID": "usd-m-list-prices"
      },
      {
        "price": 299.99,
        "pricebookID": "usd-m-sale-prices"
      }
    ]
}
```